app/assets/stylesheets/application.scss

```diff
@@ -183,15 +195,70 @@ ul {
 }
 
 .recipe-body {
+  background-color: rgba(94, 94, 94, 0.7);
+  border-style: solid;
+  border-color: #ffffff;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  border-radius: 5px;
   color: $white;
   list-style-type: none;
   margin-left: auto;
   margin-right: auto;
   text-align: left;
   width: 500px;
+  padding: 10px;
+  padding-bottom: 5px;
+}
+
+.review-body {
+  color: $white;
+  list-style-type: none;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: left;
```

Comment:
Properties should be ordered color, list-style-type, margin-left, margin-right, padding-left, padding-top, text-align, width